dcos-log/mesos/files/reader/read.go

```diff
@@ -67,9 +74,9 @@ func NewLineReader(client *http.Client, masterURL url.URL, agentID, frameworkID,
 		return nil, err
 	}
 
-	sandboxPath := fmt.Sprintf("/var/lib/mesos/slave/slaves/%s/frameworks/%s/executors/%s/runs/%s/", agentID, frameworkID, executorID, containerID)
+	sandboxPath := path.Join("/var/lib/mesos/slave/slaves/", agentID, "/frameworks/", frameworkID, "/executors/", executorID, "/runs/", containerID)
 	if taskPath != "" {
-		sandboxPath += fmt.Sprintf("tasks/%s/", taskPath)
+		sandboxPath = path.Join(sandboxPath, "tasks/"+taskPath+"/")
```

Comment:
This could be `path.Join(sandboxPath, "tasks", taskPath)` if you don't need the trailing slash.